py3status/modules/playerctl.py

```diff
@@ -273,6 +276,12 @@ def playerctl(self):
         players = []
         cached_until = self.py3.CACHE_FOREVER
         for player in tracked_players:
+            if (
+                not player.props.can_play
```

Comment:
Without digging in,  Is `not player.props.can_play` alone enough without `player_hide_non_canplay` config.... This seems like an mpris issue on chrome/chromium side.

Do chrome/chromium have this flag on `true` when it's playing?

Reply:
Actually there are other chromium based browsers like brave which has same bug. Chromium issue tracker has already bug in their system: https://issues.chromium.org/issues/40703847
Even the issue is from older raport states same issue: https://github.com/JasonLG1979/gnome-shell-extension-mpris-indicator-button/issues/10

Added some pictures about different "can" properties:

State when chromium is playing media
![image](https://github.com/user-attachments/assets/ff48f4a2-7194-484a-a38a-9693526aa8ec)

State when chromium media is paused:
![image](https://github.com/user-attachments/assets/5b97c577-7a6f-4200-aebc-9208394663fe)

State when chromium mpris is idle state:
![image](https://github.com/user-attachments/assets/2816382a-01de-4209-981a-b0108e884b2e)

VLC is opened but no media opened (just opened vlc, no media selected):
![image](https://github.com/user-attachments/assets/5c140b6b-de8c-4b7f-b6ad-25b081c3b5ed)

VLC is opened, media loaded and stopped playing media.
![image](https://github.com/user-attachments/assets/f67b3840-0472-42ee-94f7-71065751f58f)

I don't wan't to blindly hide all players with can_play == false state because I prefer to see VLC media player on statusbar just after I opened it as empty player.

It littlebit seems like hide_non_can_play is too technical so even we can simplify it to level: hide media players stopped state and user provides list on media players which he want to hide.

I searched some chromium source code and found out where this all "can" properties false comes. I suspect that line is this: https://source.chromium.org/chromium/chromium/src/+/main:components/system_media_controls/linux/system_media_controls_linux.cc;l=288

Reply:
I have been out of scene with py3status. Just sharing thoughts.

I think it doesn't benefit users to display empty players. If anything, this is maybe something that should be added in first place to ensure better quality control/behavior.

I do think `hide_non_can_play` config is too specific that partially addresses chromium-based players and not that general...  I would like an approach that that users can allow/deny (all) `can_xxx_xxx` configs (either by a dict config or new placeholders). New placeholders that exposes boolean like this... is maybe hard for users to come up with a new `format_player` that excludes those players... Dict may be be easier.

>I prefer to see VLC media player on statusbar just after I opened it as empty player.

It should be easier to just ignore players that doesn't have `can_play` flag true because of things like... You saying you don't like this empty-player thing on chrome/chromium #2255 but are okay with this on vlc... I think it would make more sense to avoid all empty-players in first place... to keep it simple.

This approach also may omit the need for adding `|{player}` at end of `format_player` config too.

Try it for a while. See if this small but simple tweak works quite well.